### PR TITLE
Fix VTX status update after saving

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes: #1671

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: src/js/tabs/vtx.js
Comment (McGiverGim) [inline src/js/tabs/vtx.js:894]: What about to include it in in the setTimeout for the save button that is just before and increase the timeout of it? At least in this way, you know that the tab will be refreshed at the same time than the button, seems more "solid" for me. 

Another solution a little more ellaborated is to use more button messages:
- The button contains "Save". The users pushes it.
- We change the text to "Saving..." and start the save process.
- In this part of code, after all has been saved, we start the initialize with a timeout of 3 seconds.
- After the timeout we change the save to "Saved" with a timeout of 2 seconds.
- After this timeout we change the text to "Save" again.

In this way is clear that we are saving and refreshing. It uses the same two timeout than now but in different order and calling one inside the other, and we only need to add the new "Saving..." message.

Comment (haslinghuis) [reply src/js/tabs/vtx.js]: I guess 4 seconds will be enough so making them the same the buttons updates more linear.

**Review by McGiverGim — APPROVED** [review]
It's not exactly what I susggested but it's ok.

**Comment by haslinghuis** [comment]
Curious to see your version as I used your functional design here.

**Review by limonspb — APPROVED** [review]

**Review by Asizon — APPROVED** [review]

**Pull request merged**
